SwrveCommonSDK/src/main/java/com/swrve/sdk/SwrveImp.java

```diff
@@ -933,33 +933,38 @@ protected void unbindAndShutdown() {
         }
     }
 
-    protected void removeCurrentDialog() {
+    protected void removeCurrentDialog(Activity callerActivity) {
```

Comment:
Ok, I didn't know that. I've googled this but can't find anything to support it. You sure?